archetype-module-content/src/test/resources/projects/basic/archetype.properties

```
@@ -0,0 +1,7 @@
+moduleName=basic
+moduleDescription=Basic Content Package Module. Useful for creating other Content Packages i.e hiv,...
+groupId=org.openmrs.content
+artifactId=content-package
+openmrsContentPackageVersion=1.0.0
+package=org.openmrs.content.content-package
+version=${package.getClass().forName("java.lang.String").getConstructor($package.getClass().forName("java.lang.String")).newInstance("1.0.0-SNAPSHOT")}
```

We don't need the weird expression here.

```suggestion
version=1.0.0-SNAPSHOT
```

---

Is this required to get the test to pass or something?